baseframe/assets.py

```diff
@@ -254,3 +254,8 @@
     'baseframe-base-bs3.css==' + __version__,
     'baseframe-networkbar.css==' + __version__,
 ]}
+
+assets['locationpicker.jquery.js'][Version('0.1.12')] = ('baseframe/js/locationpicker.jquery.js')
```

Comment:
Depends on jQuery, so needs to be `('jquery.js', 'baseframe/js/locationpicker.jquery.js')`.